index.qmd

```diff
@@ -27,23 +27,40 @@ if (any(Sys.getenv(required_env_vars) == "")) {
 list.files("R", "\\.R$", , TRUE) |> purrr::walk(source)
 server_name <- Sys.getenv("CONNECT_SERVER")
 people_path <- glue::glue("{server_name}connect/#/people/")
-dat <- get_all_users_groups()
+dat <- get_all_users_groups() |> 
+  tidyr::replace_na(list(username = "[No user]", group = "[No group]"))
 ```
 
 ## Purpose
 
-A quick lookup of [users and groups](`r people_path`) on the server `r server_name`.
+A quick lookup of [users and groups](`r people_path`) on the server `r server_name`. Find [the source on GitHub](https://github.com/The-Strategy-Unit/posit-connect-people/).
 
 ## By user
 
 ```{r}
 #| label: by-user
 dat |>
   dplyr::select(username, group) |> 
-  dplyr::arrange(username, group) |> 
+  dplyr::arrange(tolower(username), tolower(group)) |> 
+  #dplyr::mutate(dplyr::across(dplyr::where(is.factor), as.factor)) |> 

```

Comment:
Is there a reason you've left this commented code in?

Reply:
Good spot, shouldn't be there, have removed.